tools/testbed/docker/Dockerfile

```diff
@@ -0,0 +1,119 @@
+
+###############################################################################
+# Centrifuge Chain                                                            #
+# Cash on Steroids                                                            #
+#                                                                             #
+# tools/docker/Dockerfile                                                     #
+#                                                                             #
+# Handcrafted since 2020 by Centrifuge Foundation                             #
+# All rights reserved                                                         #
+#                                                                             #
+#                                                                             #
+# Description: Centrifuge chain's Docker image definition.                    #
+###############################################################################
+
+
+# -----------------------------------------------------------------------------
+# Sandbox (or compilation playground) stage
+# -----------------------------------------------------------------------------
+
+FROM rustlang/rust:nightly AS sandbox
+
+ARG ARG_RUST_TOOLCHAIN="nightly-2020-08-16"
```

Comment:
When this is hardcoded here, what effect does changing the compiler version in the settings have? Wouldn't the version be (probably) not available then?

Reply:
It's a default value. It is the value set in the "settings.mk" file which is the one used. I prefer to have a default value in case the Dockerfile is used outside the Makefile, in a manual Docker image build command (i.e. `docker image build -t repo/image_name, --target ... --file Dockerfile`).

Reply:
The design rationale behind this simple Testbed infrastructure is to provide Kraken with an easy to use Centrifuge Chain deployment script, so that they can test Centrifuge Chain transactions through the sidecar (it's a type of proxy). There's no need for the parachain.